test/mix/tasks/kitto.gen.widget_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Mix.Tasks.Kitto.Gen.WidgetTest do
+  use ExUnit.Case, async: false
+  import Kitto.MixGeneratorHelper
+
+  setup do
+    runner = fn() -> Mix.Tasks.Kitto.Gen.Widget.run(["my_widget"]) end
+    Mix.Task.clear
+    {:ok, [runner: runner]}
+  end
+
+  test "fails when widget name not provided" do
+    assert_raise Mix.Error, "No widget name provided", fn ->
+      Mix.Tasks.Kitto.Gen.Widget.run([])
+    end
+  end
+
+  test "creates widget directory", %{runner: runner} do
```

Comment:
I think we can implicitly test this by asserting that the js and the stylesheet files are created in the correct path.